internal/daemon/worker/session/manager.go

```diff
@@ -5,11 +5,19 @@ import (
 	"fmt"
 	"reflect"
 	"sync"
+	"sync/atomic"
 
 	"github.com/hashicorp/boundary/internal/errors"
 	pbs "github.com/hashicorp/boundary/internal/gen/controller/servers/services"
 )
 
+// FIXME: This is really ugly -- but not as ugly as plumbing this value into the
+// interface. We should figure out something better. For now, it will at least
+// keep sync with any changes to the value once it's initialized as the worker
+// will set it in its new function; it can also be overridden in tests if
+// desired with normal global variable caveats.
+var CloseCallTimeout = new(atomic.Int64)
```

Comment:
I like to use set/getter funcs for this sort of global vs just exporting it.  It's not a lot better... but it sure feels better.

Reply:
Fair, but will leave as an exercise for another time.